server/src/main/java/com/jungleapp/cs414/server/JungleBoard.java

```diff
@@ -12,11 +12,22 @@
     String whoseTurn; //String for player name's turn
     String fromPosition;
     String toPosition;
+    Move chosenMove;
+    Move selectedPiece;
     String errorMessage;
     boolean isActive;
     boolean createNewBoard;
 
-    public JungleBoard() { board = new Piece[9][7]; }
+    public JungleBoard() {
```

Comment:
I believe this is redundant, as ``` board = new Piece[9][7];``` already initialized the board with all null values.